qa/L0_backend_python/lifecycle/lifecycle_test.py

```diff
@@ -208,6 +208,7 @@ def test_infer_pymodel_error(self):
                         False, "Wrong exception raised or did not raise an exception"
                     )
 
+    @unittest.skip("Defer response sending test to response_sender tests")
```

Comment:
We can remove this test since we have testing for it in the response sender.

Reply:
Sure, removed.